### DISCUSSION TITLE
getShopByOwnerUserId API documentation is wrong

### DISCUSSION THREAD

**Discussion by simonj** [etsy/open-api#1152] [Q&A] [answered]
Greetings,

I'm aiming to implement functionality within my platform that retrieves information from a user's Etsy shop once they grant access. Specifically, I'm looking to use the `getShopByOwnerUserId` https://developers.etsy.com/documentation/reference/#operation/getShopByOwnerUserId endpoint from the Etsy API to obtain details such as the shop title and shop ID, based on the user ID returned by the API.

The documentation indicates that an `api_key` is sufficient for this request. However, when I test it through Postman, I encounter an error stating, `"error": "No user was provided as part of the bearer token."`

This has led to some confusion on my part regarding the correct method to access a user's Etsy shop information. Any insights or guidance would be greatly appreciated.

**Comment by Gareth064** [comment, marked as answer]
I believe, from memory, the documentation is wrong. It is an authenticated endpoint and requires a newer token to be sent with the request.

**Reply by simonj** [reply]
Yeah, i just found older posts also about that. Crazy they don't fix it.

Thanks a lot once again for your help
